### ISSUE TITLE
what is class_scale means in .json file

### ISSUE THREAD

**Issue by aslam6892** [experiencor/keras-yolo3#185]

**Comment by kabrau** [comment]
I will to explain whats I understand if you follow the code, you found these lines in yolo.py:

> xy_delta    = xywh_mask  * ... * self.xywh_scale
> wh_delta    = xywh_mask  * ... * self.xywh_scale
> conf_delta  = object_mask * ... * self.noobj_scale
> **class_delta** = object_mask * ...  *  **self.class_scale**

> loss_xy    = tf.reduce_sum(tf.square(xy_delta),       list(range(1,5)))
> loss_wh    = tf.reduce_sum(tf.square(wh_delta),       list(range(1,5)))
> loss_conf  = tf.reduce_sum(tf.square(conf_delta),     list(range(1,5)))
> **loss_class** = tf.reduce_sum(**class_delta**,               list(range(1,5)))

> **loss** = loss_xy + loss_wh + loss_conf + **loss_class**

The loss is calculated from loss_xy (box position) + loss_wh (box size) + loss_conf (precision recall) + loss_class (correct class)

Then self.class_scale has influence in class_delta/loss_class, i.e. if you change to 2, the correct class has the double importance, but if you change to 0, correct class do not has influence in the loss.
